src/master/init.h

```diff
@@ -36,35 +36,30 @@
 #include "master/personality.h"
 #include "master/topology.h"
 
-#define STR_AUX(x) #x
-#define STR(x) STR_AUX(x)
-const char id[]="@(#) version: " STR(VERSMAJ) "." STR(VERSMID) "." STR(VERSMIN) ", written by Jakub Kruszona-Zawadzki";
+/// Functions to call before normal startup
+inline const std::vector<RunTab> earlyRunTabs = {
+    RunTab{metadataserver::personality_validate, "validate personality"}};
 
-/* Run Tab */
-typedef int (*runfn)(void);
-struct run_tab {
-	runfn fn;
-	const char *name;
-};
+/// Functions to call during normal startup
+inline const std::vector<RunTab> runTabs = {
+    // has to be first
+    RunTab{hstorage_init, "name storage"},
+    // has to be second
+    RunTab{metadataserver::personality_init, "personality"},
+    RunTab{rnd_init, "random generator"},
+    // has to be before 'fs_init' and 'matoclserv_networkinit'
+    RunTab{dcm_init, "data cache manager"},
+    // has to be before 'fs_init'
+    RunTab{matoclserv_sessionsinit, "load stored sessions"},
+    RunTab{exports_init, "exports manager"},
+    RunTab{topology_init, "net topology module"},
+	// the lambda is used to select the correct fs_init overload
```

Comment:
This comment has an incorrect indentation. Please fix it.

Reply:
Done.